test/data/upgrade8to9/upgrade_verify.sh

```diff
@@ -0,0 +1,96 @@
+        }
+    ]
+}
+EOF
+
+# start osbuild-compsoer socket
+systemctl start osbuild-composer.socket
+
+# prepare a simple blueprint
+tee blueprint.toml > /dev/null << EOF
+name = "bash"
+description = "A base system with bash"
+version = "0.0.1"
+
+[[packages]]
+name = "bash"
+EOF
+
+# push and depsolve the blueprint
+composer-cli blueprints push blueprint.toml
+composer-cli blueprints despsolve bash
+
+# build a qcow image to verify functionality
+composer-cli --json compose start bash qcow2 | tee "$COMPOSE_START"
+COMPOSE_ID=$(get_build_info ".build_id" "$COMPOSE_START")
+
+# Wait for the compose to finish.
+while true; do
+    composer-cli --json compose info "${COMPOSE_ID}" | tee "$COMPOSE_INFO" > /dev/null
+    COMPOSE_STATUS=$(get_build_info ".queue_status" "$COMPOSE_INFO")
+
+    # Is the compose finished?
+    if [[ $COMPOSE_STATUS != RUNNING ]] && [[ $COMPOSE_STATUS != WAITING ]]; then
+        break
+    fi
+
+    # Wait 30 seconds and try again.
+    sleep 30
+done
+
+# if the compose succeds consider the test pass
+if [[ $COMPOSE_STATUS != FINISHED ]]; then
+    echo "Something went wrong with the compose. 😢"
```

Comment:
In case this happens do we get any logs ?

Reply:
No nothing really, I'll add something.

Reply:
I'm getting metadata and logs like we do in other tests and upload them as artifacts.